rspace/src/main/scala/coop/rchain/rspace/history/LMDBTrieStore.scala

```diff
@@ -0,0 +1,63 @@
+package coop.rchain.rspace.history
+
+import java.nio.ByteBuffer
+
+import coop.rchain.rspace.LMDBStore
+import coop.rchain.shared.AttemptOps._
+import org.lmdbjava.DbiFlags.MDB_CREATE
+import org.lmdbjava.{Dbi, Env, Txn}
+import scodec.Codec
+import scodec.bits.BitVector
+
+class LMDBTrieStore[K, V] private (val env: Env[ByteBuffer], _dbTrie: Dbi[ByteBuffer])(
+    implicit
+    codecK: Codec[K],
+    codecV: Codec[V])
+    extends ITrieStore[Txn[ByteBuffer], K, V] {
+
+  private[rspace] def createTxnRead(): Txn[ByteBuffer] = env.txnRead
```

Comment:
This could be moved to a base class for both `LMDB` implementations to avoid duplication. Same applies to both methods below

Reply:
Yes, I'm aware of this and I mentioned this in my notes [above](https://github.com/rchain/rchain/pull/757#issue-189504157).  Part of my intent here is to keep this work for separate from the current store for this stage of development.  

I'm totally down to approach the question of how its factored together with the existing store once this work has stabilized.